lib/imlib/framebuffer.c

```diff
@@ -95,34 +95,27 @@ static void jpegbuffer_init_from_image(image_t *img) {
 }
 
 void framebuffer_update_jpeg_buffer(image_t *src) {
-    static int overflow_count = 0;
+    static int overflow_count = 0, copy_overflow_count = 0;
```

Comment:
The IDE can count that.

Reply:
Changed it to a simpler system on the camera.